src/transformers/modeling_tf_utils.py

```diff
@@ -102,6 +101,18 @@
 ]
 
 
+def expand_1d(data):
```

Comment:
This function was part of the Keras code that is not really in the public API. I just copied the whole thing rather than trying to import a moving target.

Reply:
Should be in `tf_utils` though.

Reply:
Moved!